### DISCUSSION TITLE
Highlight

### DISCUSSION THREAD

**Discussion by albersonmiranda** [chipzoller/hugo-clarity#182] [Q&A]
If hightlight style is hardcoded into _syntax.sass, what does 
```
  [highlight]
    style = "whatever"
```
do?

And more important, how do I change highlight theme?

**Comment by onweru** [comment]
@albersonmiranda, that setting isn't applicable to this theme. If you need to change the highlighting looks you would need to [edit colors directly via file](https://github.com/chipzoller/hugo-clarity/blob/7464d0088f20d3d77ab9a0cf5762535e15cc6e40/assets/sass/_syntax.sass#L109-L170). It's advised that you [override those styles](https://github.com/chipzoller/hugo-clarity#custom-css-and-js)
